docs/content/stable/explore/ysql-language-features/advanced-features/partitions.md

```diff
@@ -125,46 +160,33 @@ CREATE TABLE order_changes_2021_02 PARTITION OF order_changes
 
 Note the following:
 
-- The primary key for a partitioned table should always contain the partition key. In the current release of YugabyteDB, it is recommended to set the primary key directly on partitions.
+- The primary key for a partitioned table should always contain the partition key.
 - If you choose to define row triggers, you do so on individual partitions instead of the partitioned table.
-- A partition table does not inherit tablespaces from its parent. A partition table by default is placed according to cluster configuration.
+- Creating a foreign key reference on a partitioned table is not supported.
+- A partition table inherits tablespaces from its parent.
 - You cannot mix temporary and permanent relations in the same partition hierarchy.
 - If you have a default partition in the partitioning hierarchy, you can add new partitions only if there is no data in the default partition that matches the partition constraint of the new partition.
 
-## Partition Pruning
+## Partition Pruning and Constraint Exclusion
```

Comment:
We prefer sentence case in headings. For a small addition like this, just follow whatever's in the rest of the file...
```suggestion
## Partition pruning and constraint exclusion
```